script.js

```diff
@@ -0,0 +1,690 @@
+  symbol.classList.add(cardInfo.display, cardInfo.color, 'suit', 'card-corner-suit', 'card-topleft', 'card__face', 'card__face--front');
+  symbol.innerText = cardInfo.symbol;
+
+  const name = document.createElement('div');
+  name.classList.add(cardInfo.display, cardInfo.color, 'name', 'card-corner-rank', 'card-topleft', 'card__face', 'card__face--front');
+  name.innerText = cardInfo.name;
+
+  // creation of back of card element
+  const cardBack = document.createElement('img');
+  cardBack.setAttribute('src', 'Assets/cardback.png');
+  cardBack.setAttribute('width', '300');
+  cardBack.setAttribute('height', '300');
+  cardBack.classList.add('card__face', 'card__face--back');
+
+  const card = document.createElement('div');
+  card.classList.add('card');
+  card.setAttribute('id', 'card');
+  card.appendChild(name);
+  card.appendChild(symbol);
+  card.appendChild(cardBack);
+  return card;
+};
+
+// CSS animation for card Flip
+const animationDiv = document.createElement('div');
+animationDiv.classList.add('scene');
+
+// cardFlip.appendChild(cardBack1);
+// animationDiv.appendChild(cardFlip);
+cardContainer.appendChild(animationDiv);
+
+const tradeCards = () => {
+  // trade the selected cards in playerhand
+  for (let i = 0; i < playerHand.length; i += 1) {
+    for (let j = 0; j < selectedCardsArray.length; j += 1) {
+      if (selectedCardsArray[j].rank === playerHand[i].rank
+        && selectedCardsArray[j].suit === playerHand[i].suit) {
+        playerHand.splice(i, 1, deck.pop());
+      }
+    }
+  }
+  console.log(playerHand);
+  // empty the cards in trade array after the trade is done
+  selectedCardsArray = [];
+  // clear display of current player's hand
+  cardContainer.innerHTML = '';
+  for (let i = 0; i < playerHand.length; i += 1) {
+    const cardEl = createCard(playerHand[i]);
+    cardContainer.appendChild(cardEl);
+  }
+};
+
+// appending child order/display order
+mainDisplayDiv.appendChild(bidAmountLabel);
+mainDisplayDiv.appendChild(bidAmountInput);
+mainDisplayDiv.appendChild(brEL);
+mainDisplayDiv.appendChild(brEL.cloneNode());
+mainDisplayDiv.appendChild(dealButton);
+mainDisplayDiv.appendChild(brEL.cloneNode());
+mainDisplayDiv.appendChild(cardContainer);
+// store the cards based on rank, each rank has a different row in the array
+let rankHandArray = [[]];
+
+const arrangeCards = () => {
+  // for each position of the playerhand starting from the 0th index
+  // find higher rank cards and swap them in those position
+  let j = 1;
+  for (let i = 0; i < (playerHand.length - 1); i += 1) {
+    for (let k = j; k < playerHand.length; k += 1) {
+      if (playerHand[k].rank > playerHand[i].rank) {
+        const lowerRankCard = playerHand[i];
+        const higherRankCard = playerHand[k];
+        playerHand[i] = higherRankCard;
+        playerHand[k] = lowerRankCard;
+      }
+    }
+    j += 1;
+  }
+};
+const groupRankedPlayerCards = () => {
+  // empty hand from previous games
+  // playerHand[0].rank = 13;
```

Comment:
Same comment as above on commented-out code

Reply:
This was for me to test and will be removed from the final code.